README.md

```diff
@@ -32,6 +32,8 @@ https://github.com/Mr0grog/editorconfig-textmate/releases/latest
 
 Un-tar it and double-click the `editorconfig-textmate.tmplugin` file to install. You can also manually drag the `.tmplugin` package into `~/Library/Application Support/TextMate/PlugIns/` in Finder.
 
+**Note:** The PlugIn is loaded after restarting TextMate and must not be activated manually.
+It will not be listed in the Bundles list since Bundle != PlugIn in TextMate.
```

Comment:
Good point. I wrote it because I was looking for a way to manually activate the plugin (which is necessary in most other plugin infrastructures). I updated this with minor adjustments to your version.